shell/platform/windows/text_input_plugin_unittest.cc

```diff
@@ -97,21 +102,63 @@ static std::unique_ptr<rapidjson::Document> EncodedEditingState(
   return arguments;
 }
 
-class MockTextInputPluginDelegate : public TextInputPluginDelegate {
+class MockFlutterWindowsView : public FlutterWindowsView {
  public:
-  MockTextInputPluginDelegate() {}
-  virtual ~MockTextInputPluginDelegate() = default;
+  MockFlutterWindowsView(std::unique_ptr<WindowBindingHandler> window)
+      : FlutterWindowsView(std::move(window)) {}
+  virtual ~MockFlutterWindowsView() = default;
 
   MOCK_METHOD(void, OnCursorRectUpdated, (const Rect&), (override));
   MOCK_METHOD(void, OnResetImeComposing, (), (override));
 
  private:
-  FML_DISALLOW_COPY_AND_ASSIGN(MockTextInputPluginDelegate);
+  FML_DISALLOW_COPY_AND_ASSIGN(MockFlutterWindowsView);
 };
 
 }  // namespace
 
-TEST(TextInputPluginTest, TextMethodsWorksWithEmptyModel) {
+class TextInputPluginTest : public WindowsTest {
+ public:
+  TextInputPluginTest() = default;
+  virtual ~TextInputPluginTest() = default;
+
+ protected:
+  FlutterWindowsEngine* engine() { return engine_.get(); }
+  MockFlutterWindowsView* view() { return view_.get(); }
+  MockWindowBindingHandler* window() { return window_; }
+
+  void use_headless_engine() {
```

Comment:
Use `UpperCamelCase` for methods. For simple accessors and mutators, you can use the same naming as variables (e.g., `engine` and `set_engine`).

Ref: https://google.github.io/styleguide/cppguide.html#Function_Names

Reply:
Fixed this and the other areas where I stole this name from :)